### PR TITLE
Lesson006

### PR DESCRIPTION
**Pull request by DenegaSergey**

### CODE REVIEW

File: Lesson006/Task 1 HW06.TheTriangleBuilder/Program.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+            for (int lines = userInput; lines >= 1; lines--)
+            {
+                for (int spaces = userInput - 1; spaces >= lines; spaces--)
+                {
+                    Console.Write(" ");
+                }
+
+                for (int colums = 1; colums <= lines; colums++)
+                {
```
Comment (Onix-one) [inline Lesson006/Task 1 HW06.TheTriangleBuilder/Program.cs]: Inside an expression inside a loop, it's common to use i, j or k. For example var i = 0; i < array.length; i++

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson006/Task 2 HW06.ThirdArrayCreator/Program.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            int[] arrayKeyboardInput = new int[5];
+
+            Console.WriteLine("Keyboard input: ");
+
+            for (int i = 0; i < arrayKeyboardInput.Length; i++)
+                arrayKeyboardInput[i] = int.Parse(Console.ReadLine());
+
+            int[] sumOfArchives = new int[5];
+            for (int i = 0; i < 5; ++i)
+            {
```
Comment (Onix-one) [inline Lesson006/Task 2 HW06.ThirdArrayCreator/Program.cs]: What is the number 5? I don't understand. Use the array length or create const.

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson006/Task 3 HW06.ArrayModifier/Program.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+            for (int i = 0; i < arrayRandomInput.Length; i++)
+            {
+                arrayRandomInput[i] = random.Next(20);
+                Console.WriteLine("Array element value " +  " = " + arrayRandomInput[i]);
+            }
+
+            int[] arrayKeyboardInput = new int[5];
+
+            Console.WriteLine("Keyboard input: ");
```
Comment (Onix-one) [inline Lesson006/Task 3 HW06.ArrayModifier/Program.cs]: This task is the same as the previous task. I don't see solve task ArrayModifier.

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson006/Task 3 HW06.ArrayModifier/Program.cs (hunk 3 above)
Comment (DenegaSergey) [reply Lesson006/Task 3 HW06.ArrayModifier/Program.cs]: Changed Task3

File: Lesson006/Task 1 HW06.TheTriangleBuilder/Program.cs (hunk 1 above)
Comment (DenegaSergey) [reply Lesson006/Task 1 HW06.TheTriangleBuilder/Program.cs]: Fixed

File: Lesson006/Task 2 HW06.ThirdArrayCreator/Program.cs (hunk 2 above)
Comment (DenegaSergey) [reply Lesson006/Task 2 HW06.ThirdArrayCreator/Program.cs]: Corrected

File: Lesson006/Task 3 HW06.ArrayModifier/Program.cs [hunk 4]
```diff
@@ -1,39 +1,45 @@
...
+            lastNumber = Convert.ToInt32(Console.ReadLine());
+
+            myArrey[myArrey.Length - 1] = lastNumber;
+
+            int rightShift = myArrey[myArrey.Length - 1];
+
+            for (int i = myArrey.Length - 1; i > 0; i--) //How to rotate an array to the left?
+            {
+                myArrey[i] = myArrey[i - 1];
+            }
```
Comment (Onix-one) [inline Lesson006/Task 3 HW06.ArrayModifier/Program.cs]: You have to complete part of the task. To shift the elements, we need to place the values starting from the specified index into a new array and then reassign starting from the new index. This is one of the solutions.

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson006/Task 3 HW06.ArrayModifier/Program.cs (hunk 4 above)
Comment (DenegaSergey) [reply Lesson006/Task 3 HW06.ArrayModifier/Program.cs]: Fixed the solution

**Pull request merged**